tests/executables_src/testUnifiedTypeChangingDecorator.cpp

```diff
@@ -61,7 +65,8 @@ class DecoratorBackend : public ExceptionDummy {
     auto [path, type] = getPathAndType(registerPathName);
 
     return getTypeChangingDecorator<UserType>(
-        ExceptionDummy::getRegisterAccessor_impl<float>(path, numberOfWords, wordOffsetInRegister, flags), type);
+        CALL_BASE_FUNCTION_TEMPLATE(getRegisterAccessor_impl, float, path, numberOfWords, wordOffsetInRegister, flags),
```

Comment:
What if I want to call NumericAddressedBackend::getRegisterAccessor_impl() or DummyBackend::getRegisterAccessor_impl() here?